lib/inc/drogon/utils/Utilities.h

```diff
@@ -45,8 +46,10 @@ std::string hexToBinaryString(const char *ptr, size_t length);
 std::vector<char> hexToBinaryVector(const char *ptr, size_t length);
 
 /// Split the string into multiple separated strings.
-std::vector<std::string> splitString(const std::string &str,
-                                     const std::string &separator);
+inline auto splitString = std::bind(&trantor::splitString,
```

Comment:
I'm not sure, but it may be slower than a normal inline function. see [this](https://www.youtube.com/watch?v=ZlHi8txU4aQ) 
And users have to take a look into trantor to get the parameter types and the return type.

Reply:
I see. Then lets do a simple wrap